device/celestica/x86_64-cel_seastone-r0/Seastone-DX010-25-50/th-seastone-dx010-96x25G-16x50G.config.bcm

```diff
@@ -0,0 +1,916 @@
+phy_xaui_rx_polarity_flip_111=0x0001
+phy_xaui_tx_polarity_flip_112=0x0001
+phy_xaui_rx_polarity_flip_112=0x0001
+phy_xaui_tx_polarity_flip_113=0x0001
+phy_xaui_rx_polarity_flip_113=0x0001
+
+#WC27
+phy_xaui_tx_polarity_flip_114=0x0001
+phy_xaui_rx_polarity_flip_114=0x0001
+phy_xaui_tx_polarity_flip_115=0x0001
+phy_xaui_rx_polarity_flip_115=0x0001
+phy_xaui_tx_polarity_flip_116=0x0001
+phy_xaui_rx_polarity_flip_116=0x0001
+phy_xaui_tx_polarity_flip_117=0x0001
+phy_xaui_rx_polarity_flip_117=0x0000
+
+#WC0
+phy_xaui_tx_polarity_flip_1=0x0001
+phy_xaui_rx_polarity_flip_1=0x0001
+phy_xaui_tx_polarity_flip_2=0x0001
+phy_xaui_rx_polarity_flip_2=0x0001
+phy_xaui_tx_polarity_flip_3=0x0000
+phy_xaui_rx_polarity_flip_3=0x0001
+phy_xaui_tx_polarity_flip_4=0x0000
+phy_xaui_rx_polarity_flip_4=0x0001
+
+#WC1
+phy_xaui_tx_polarity_flip_5=0x0001
+phy_xaui_rx_polarity_flip_5=0x0000
+phy_xaui_tx_polarity_flip_6=0x0001
+phy_xaui_rx_polarity_flip_6=0x0000
+phy_xaui_tx_polarity_flip_7=0x0001
+phy_xaui_rx_polarity_flip_7=0x0000
+phy_xaui_tx_polarity_flip_8=0x0000
+phy_xaui_rx_polarity_flip_8=0x0000
+
+#WC2
+phy_xaui_tx_polarity_flip_9=0x0000
+phy_xaui_rx_polarity_flip_9=0x0000
+phy_xaui_tx_polarity_flip_10=0x0001
+phy_xaui_rx_polarity_flip_10=0x0000
+phy_xaui_tx_polarity_flip_11=0x0000
+phy_xaui_rx_polarity_flip_11=0x0000
+phy_xaui_tx_polarity_flip_12=0x0000
+phy_xaui_rx_polarity_flip_12=0x0001
+
+#WC3
+phy_xaui_tx_polarity_flip_13=0x0001
+phy_xaui_rx_polarity_flip_13=0x0000
+phy_xaui_tx_polarity_flip_14=0x0001
+phy_xaui_rx_polarity_flip_14=0x0000
+phy_xaui_tx_polarity_flip_15=0x0001
+phy_xaui_rx_polarity_flip_15=0x0000
+phy_xaui_tx_polarity_flip_16=0x0001
+phy_xaui_rx_polarity_flip_16=0x0000
+
+#WC4
+phy_xaui_tx_polarity_flip_17=0x0003
+phy_xaui_rx_polarity_flip_17=0x0000
+phy_xaui_tx_polarity_flip_18=0x0001
+phy_xaui_rx_polarity_flip_18=0x0000
+
+#WC5
+phy_xaui_tx_polarity_flip_21=0x0000
+phy_xaui_rx_polarity_flip_21=0x0000
+phy_xaui_tx_polarity_flip_22=0x0000
+phy_xaui_rx_polarity_flip_22=0x0000
+
+#WC6
+phy_xaui_tx_polarity_flip_25=0x0002
+phy_xaui_rx_polarity_flip_25=0x0001
+phy_xaui_tx_polarity_flip_26=0x0000
+phy_xaui_rx_polarity_flip_26=0x0001
+
+#WC7
+phy_xaui_tx_polarity_flip_29=0x0002
+phy_xaui_rx_polarity_flip_29=0x0000
+phy_xaui_tx_polarity_flip_30=0x0000
+phy_xaui_rx_polarity_flip_30=0x0000
+
+#WC28
+phy_xaui_tx_polarity_flip_118=0x0003
+phy_xaui_rx_polarity_flip_118=0x0003
+phy_xaui_tx_polarity_flip_119=0x0003
+phy_xaui_rx_polarity_flip_119=0x0003
+
+#WC29
+phy_xaui_tx_polarity_flip_122=0x0000
+phy_xaui_rx_polarity_flip_122=0x0000
+phy_xaui_tx_polarity_flip_123=0x0001
+phy_xaui_rx_polarity_flip_123=0x0000
+
+#WC30
+phy_xaui_tx_polarity_flip_126=0x0003
+phy_xaui_rx_polarity_flip_126=0x0000
+phy_xaui_tx_polarity_flip_127=0x0003
+phy_xaui_rx_polarity_flip_127=0x0000
+
+#WC31
+phy_xaui_tx_polarity_flip_130=0x0002
+phy_xaui_rx_polarity_flip_130=0x0000
+phy_xaui_tx_polarity_flip_131=0x0001
+phy_xaui_rx_polarity_flip_131=0x0000
```

Comment:
no serdes settings?

Reply:
The default worked for our cable length.  For different cable lengths, we should use bcmcmd or SAI APIs to change the settings dynamically.